src/smolagents/models.py

```diff
@@ -249,7 +251,7 @@ def get_tool_call_chat_message_from_text(text: str, tool_name_key: str, tool_arg
         content=text,
         tool_calls=[
             ChatMessageToolCall(
-                id=uuid.uuid4(),
+                id=str(uuid.uuid4()),
```

Comment:
@aymeric-roucher 

I guess this is a regression. Otherwise `parse_json_blob` would crash since model in the output would include:
```
{"id": UUID("f37b1f32-29b5-45d4-ace9-a4ab347802a3")}
```
That `json.loads` can't handle.

Reply:
Moved this into a separate PR to speed the things up https://github.com/huggingface/smolagents/pull/990